monolithe/generators/lang/javascript/config/config.json

```diff
@@ -1,5 +1,5 @@
 {    
-    "base_attrs": ["entityScope", "externalID", "lastUpdatedBy"],
+    "base_attrs": ["entityScope", "externalID", "lastUpdatedBy", "creationDate", "lastUpdatedDate", "owner"],
```

Comment:
the naming of this key is incorrect I have to juggle my memory every time I review, these are non-modifiable attributes, can you please add at least some comments to let devs know what are these used for?